modernomad/settings/common.py

```diff
@@ -33,6 +33,7 @@
 # Local time zone for this installation. Choices can be found here:
 # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
 TIME_ZONE = 'America/Los_Angeles'
+DATE_FORMAT = '%m/%d/%Y'
```

Comment:
Maybe this is opportune time to switch to a not terrible date format. ;)

I vote for ISO `%Y-%m-%d`